x-pack/plugins/index_lifecycle_management/public/application/sections/policy_list/components/policy_table.tsx

```diff
@@ -103,6 +104,17 @@ export const PolicyTable: React.FunctionComponent<Props> = ({ policies }) => {
   );
   const { setListAction } = usePolicyListContext();
 
+  const PAGE_SIZE_OPTIONS = [10, 25, 50];
```

Comment:
It is probably best to keep this constant outside of the component, next to `SHOW_MANAGED_POLICIES_BY_DEFAULT` for example.

Reply:
Addressed in [096abab](https://github.com/elastic/kibana/pull/186997/commits/096ababced9dd39a51756bebea3b67b1dc95650b).